src/SDKs/Batch/DataPlane/Azure.Batch/ImageReference.cs

```diff
@@ -0,0 +1,37 @@
+﻿namespace Microsoft.Azure.Batch
```

Comment:
@matthchr no license header in this file

Reply:
Will fix